GitUI/CommandsDialogs/FormCherryPick.cs

```diff
@@ -126,9 +126,9 @@ private void Revert_Click(object sender, EventArgs e)
 
                 // Don't verify whether the command is successful.
                 // If it fails, likely there is a conflict that needs to be resolved.
-                FormProcess.ShowDialog(this, process: null, arguments: command, Module.WorkingDir, input: null, useDialogSettings: true);
+                FormProcess.ShowDialog(Owner, process: null, arguments: command, Module.WorkingDir, input: null, useDialogSettings: true);
 
-                MergeConflictHandler.HandleMergeConflicts(UICommands, this, AutoCommit.Checked);
+                MergeConflictHandler.HandleMergeConflicts(UICommands, Owner, AutoCommit.Checked);
```

Comment:
It is not obvious to me why this time, the `Owner` must be used instead of `this` in contrast to 26361af.
FormCherryPick is visible as long as the git command is run.

If this change is correct, it must be applied to the MessageBox in line 109, too.